awslambdaric/bootstrap.py

```diff
@@ -324,7 +324,7 @@ class FramedTelemetryLogSink(object):
 
     def __init__(self, fd):
         self.fd = int(fd)
-        self.frame_type = 0xA55A0001 .to_bytes(4, "big")
+        self.frame_type = 0xA55A0001.to_bytes(4, "big")
```

Comment:
CI tests are failing, suspicious that they might be related to this change

Reply:
Fixed. It was unrelated, the failing format check hid the failing test.